scrabble.js

```diff
@@ -1,16 +1,202 @@
 
 const Scrabble = {
+
+
   score(word) {
 
+    let wordScore = 0
+
+    const SCORING_RUBRIK = {
+      'a': 1,
+      'e': 1,
+      'i': 1,
+      'o': 1,
+      'u': 1,
+      'l': 1,
+      'n': 1,
+      'r': 1,
+      's': 1,
+      't': 1,
+      'd': 2,
+      'g': 2,
+      'b': 3,
+      'c': 3,
+      'm': 3,
+      'p': 3,
+      'f': 4,
+      'h': 4,
+      'v': 4,
+      'w': 4,
+      'y': 4,
+      'k': 5,
+      'j': 8,
+      'x': 8,
+      'q': 10,
+      'z': 10
+    }
+    if (typeof word == 'number'){
+      throw 'Letters only please.';
+    }
+
+    for (let i = 0; i < word.length; i++) {
+
+      let letter = word[i];
+
+      if (!RegExp(/^[a-zA-Z]+$/).test(letter)){
+
+        throw `Letters only please. Letter: ${letter} i == ${i} word: ${word}`
+      }
```

Comment:
It would be a little faster to test this regular expression against the whole word outside the loop.